qiskit_ibm_runtime/base_primitive.py

```diff
@@ -56,24 +56,13 @@ class BasePrimitiveV2(ABC, Generic[OptionsT]):
 
     def __init__(
         self,
-        backend: Optional[Union[str, BackendV1, BackendV2]] = None,
-        session: Optional[Session] = None,
+        mode: Optional[Union[BackendV1, BackendV2, Session, Batch]] = None,
```

Comment:
We can't just drop `session` and `backend`. They need to be deprecated first. There are utility [functions](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/utils/deprecation.py) that you can use to issue the standard deprecation warning.

Reply:
I deprecated them on Sampler and Estimator, this is the base class. Do I need to deprecate also here?

Reply:
oh my apologies, I didn't scroll down far enough to see that!